src/openforms/js/components/admin/form_design/RegistrationFields.js

```diff
@@ -76,10 +89,51 @@ const RegistrationFields = ({
     <Fieldset
       style={{'--of-add-another-text': `"${addAnotherMsg}"`}}
       extraClassName="admin-fieldset"
+      title={
+        <>
+          {backendName || backendKey}
+          <DeleteIcon onConfirm={onDelete} />
+        </>
+      }
     >
+      {
+        // <FormRow>
+        //   <Field
+        //     name={`form.registrationBackends.${index}.key`}
+        //     label={<FormattedMessage defaultMessage="Key" description="Registration backend key" />}
+        //     helpText={
+        //       <FormattedMessage
+        //         defaultMessage="The key to use to refer to this configuration in form logic."
+        //         description="Backend key help text"
+        //       />
+        //     }
+        //     disabled
+        //   >
+        //   <TextInput onChange={onChange} value={backendKey} />
+        //   </Field>
+        // </FormRow>
+        //
```

Comment:
Yes initially I had this as an input, because I had the action interface use logic that resolved to the key. So having it user defined was nicer. But no more...